soroban-sdk/src/env.rs

```diff
@@ -408,7 +410,7 @@ impl Env {
     {
         let rv = internal::Env::try_call(
             self,
-            contract_id.to_object(),
+            contract_id.contract_id().to_object(),
```

Comment:
TODO: Catch the error when Address is not a contract. This line is a problem. Try_invoke should never panic, but the call to contract_id() does panic now. I probably should have kept contract_id() as an Option<>. I'll change it back and massage the None into a general error of sorts.

Reply:
Fixed in [c02d267](https://github.com/stellar/rs-soroban-sdk/pull/947/commits/c02d2671096a9b475cb6ec0b72faa0389622a961)